README.md

```diff
@@ -22,24 +22,24 @@ To quickly get started, follow the installation instructions for [Python][1] or
+| `addLayers`            | Whether to install the Datadog Lambda library as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]).                                                                                                          |
+| `addExtension`         | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, it's required to set the `apiKey` (or `apiKMSKey`) parameter. While using `addExtension` leave out `forwarderArn`. The Datadog Lambda Extension Layer is in public preview. Learn more about the Lambda Extension Layer [here][8].                                                                                   |
+| `logLevel`             | The log level, set to `DEBUG` for extended logging. Defaults to `info`.                                                                                                                                                                                                                                                                                                                                             |
+| `enableXrayTracing`    | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                                       |
+| `enableDDTracing`      | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                                  |
+| `enableApiGatewayLogs` | Enable automatic subscription of the Datadog Forward to API Gateway log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Datadog Forward -> Datadog Forwarder

Also, I'm not sure that `enableApiGatewayLogs` is the best name for this setting. My first thought when I hear it is that this would enable/disable the logs. However, it sounds like the real behavior of this setting is to enable/disable **subscribing the Forwarder** to the API Gateway logs. Maybe it should be called something like `subscribeForwarderToApiGatewayLogs`?

Finally, it seems like this will only work if a `forwarderArn` is provided. We should say that and also probably add some kind of check that will raise an error if you try to enable subscribing to API Gateway logs without providing a forwarder ARN.